tf/src/tf_echo.cpp

```diff
@@ -91,14 +91,25 @@ int main(int argc, char ** argv)
   }
   ros::Rate rate(rate_hz);
 
+  int precision(3);
+  {
+    ros::NodeHandle p_nh("~");
+    p_nh.param("precision", precision, 3);
+    if (precision < 1)
+    {
+      std::cerr << "Precision must be > 0\n";
+      return -1;
+    }
+    ROS_INFO_STREAM("Precision: " << precision);
```

Comment:
Please update this to only print if it's overridden from the default and mention that it's being overridden by the parameter.

Reply:
Ok